src/GrafanaDashboardDeploy/src/DotNet.Grafana.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.4" />
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.3.1" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="16.4.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.8">
```

Comment:
...FxCop? That's usually a thing we avoid.  Where is this reference coming from? I don't recall seeing it in any other project.

Reply:
It's just [static code analysis](https://docs.microsoft.com/en-us/visualstudio/code-quality/roslyn-analyzers-overview?view=vs-2019). VS recommends it from the project settings. Its complaints seemed reasonable. What's the history with FxCop?

Reply:
It's not a functional dependency so I'll just remove it here.

Reply:
Maybe it's fine? It looks like it's just a weirdly named package that actually comes from "dotnet/roslyn-analyzers". It's just normal analyzers with "FxCop" slapped on the name.  Since it's not the actual FxCop tool, it's probably fine.  But I don't like random projects having different coding styles/analyzers than others.  We should either decide it's useful and use it everywhere (putting it in directory.build.props), or not and use it nowhere.